spec/views/pages/homepage.html.erb_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe "pages/homepage.html.erb", type: :view do
+
```

Comment:
you can remove this empty line here, and on line 21, like the spacing between the specs (line 10, 15) a lot!